integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/registry/config/filters/dynamic_seq1.xml

```diff
@@ -1,5 +1,5 @@
 <!--
-  ~  Copyright (c) 2005-2010, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+  ~  Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
We should not update the license if it is a existing file

Reply:
This was moved from tests-mediator-1/src/test/resources/artifacts/ESB/synapseconfig/filters/conditional_router/dynamic_seq1.xml to tests-mediator-1/src/test/resources/artifacts/ESB/server/registry/config/filters/dynamic_seq1.xml.

Reply:
but still the same file with original content just restructuring.